src/Components/Main/Main.scss

```diff
@@ -42,17 +42,135 @@
         .header--item {
             margin: 10px 0px;
             cursor: pointer;
+            // position: relative;
             color: rgba(white, 0.8);
             transition: transform 0.2s ease-in-out;
-
+           
             &:hover {
                 color: rgba(white, 1);
-                transform: scale(1.1);
+                transform: scale(1.2);
             }
+          
+            
         }
+        .tooltip{
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            text-align: center;
+            border-radius: 8px;
+            width: 5vw;
+            height: 5vh;
+            position: absolute;
+            top: 0;
+            left: 140%;
+            color: black;
+            background-color: white;
+            opacity: 0;
+        }
+        .tooltip .tooltiptext::after{
+            content: "";
+            position: absolute;
+            top: 30%;
+            left: 0;
+            margin-left: -5px;
+            border-width: 5px;
+            border-style: solid;
+            border-color: transparent white white transparent;
+          }
+        .tooltip .tooltiptext::before{
+            content: "";
+            position: absolute;
+            bottom: 17%;
+            left: 0;
+            margin-left: -5px;
+            border-width: 5px;
+            border-style: solid;
+            border-color: white white transparent transparent;
+          }
+        .share{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
 
-        .shareIcon--active {
+            }
+
+        }
+        .brush{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
+        .eraser{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
+        .rectangle{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
+        .circle{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
+        .clear{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
 
+            }
+
+        }
+        .undo{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
+        .redo{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
+        .zoomIn{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
+        .zoomOut{
+            position: relative;
+            &:hover .tooltip{
+                opacity: 1;
+
+            }
+
+        }
```

Comment:
Why use a different class for different if it has the same styling. Can we use a single class here please and also update it in the js file?

Reply:
@vedant-jain03 Should I give same class to Zoom In and Zoom Out? That's what you want to say.

Reply:
Not actually, use the class as "tooltip_component_text" for every class and give a single styling for it. In this way we can reduce the line of codes and make use of class functionality.